app/grandchallenge/core/templatetags/negate.py

```diff
@@ -0,0 +1,8 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def negate(value):
```

Comment:
Why do we need this? I would have thought that doing `not value` in the template would work? Or if you need it as a variable using `with`?

Reply:
I tried both, and there does not seem to be a construct that works. So, I resorted to this.

Reply:
Quickly double checked and yeah, none of these work:

`{% with hide=not form.initial.parent %}`
`{% with not form.initial.parent as hide %}`

I think (but we no longe need it) we could have used `|default:True` instead. Which isn't much easier to read imho.